extensions/adobe/experience/campaign/experienceevent.schema.json

```diff
@@ -191,6 +191,16 @@
               "type": "integer",
               "description":
                 "The delivery template's ID used to initialize this delivery.\n\nThe type the template used in Adobe Campaign can be identified using the `messageClass`.\n\n* `messageClass` = `oneTime`: the template is standard delivery template.\n* `messageClass` = `continuous`: the template is a recurring delivery.\n* `messageClass` = `transactional`: the template is a transactionnal message template.\n"
+            },
+            "xdm:deliveryLabel": {
+              "title": "Delivery Label",
+              "type": "string",
+              "description" : "Human friendly name of the campaign activity which is originating this message."
+            },
+            "xdm:deliveryName": {
+              "title": "Delivery Internal name",
+              "type": "string",
+              "description" : "Friendly identifier of the campaign activity which is originating this message."
```

Comment:
I think the topic is that both the `xdm:deliveryLabel` and `xdm:deliveryName` are intended to be _human readable_. I think they should be consistently described as such.

Reply:
done